clang/docs/HLSL/ExpectedDifferences.rst

```diff
@@ -119,3 +119,49 @@ behavior between Clang and DXC. Some examples include:
   diagnostic notifying the user of the conversion rather than silently altering
   precision relative to the other overloads (as FXC does) or generating code
   that will fail validation (as DXC does).
+
+Correctness improvements and bug fixes
+======================================
+
+Entry point functions & ``static`` keyword
+------------------------------------------
+Marking a shader entry point function ``static`` will result in a error.
```

Comment:
This is currently not the case. I have filed an issue to consistently report an error when a shader entry point is not found or it is marked `static`: llvm/llvm-project#101571